AccessibilitySnapshot/SnapshotTesting/Classes/SnapshotTesting+Accessibility.swift

```diff
@@ -0,0 +1,187 @@
+
+    /// Snapshots the current view with colored overlays of each accessibility element it contains, as well as an
+    /// approximation of the description that VoiceOver will read for each element.
+    public static var accessibilityImage: Snapshotting {
+        return .accessibilityImage()
+    }
+
+    /// Snapshots the current view with colored overlays of each accessibility element it contains, as well as an
+    /// approximation of the description that VoiceOver will read for each element.
+    ///
+    /// - parameter showActivationPoints: When to show indicators for elements' accessibility activation points.
+    /// Defaults to showing activation points only when they are different than the default activation point for that
+    /// element.
+    /// - parameter useMonochromeSnapshot: Whether or not the snapshot of the `view` should be monochrome. Using a
+    /// monochrome snapshot makes it more clear where the highlighted elements are, but may make it difficult to
+    /// read certain views. Defaults to `true`.
+    /// - parameter markerColors: The array of colors which will be chosen from when creating the overlays
+    public static func accessibilityImage(
+        showActivationPoints activationPointDisplayMode: ActivationPointDisplayMode = .whenOverridden,
+        useMonochromeSnapshot: Bool = true,
+        markerColors: [UIColor] = []
+    ) -> Snapshotting {
+        guard isRunningInHostApplication else {
+            fatalError("Accessibility snapshot tests cannot be run in a test target without a host application")
+        }
+
+        return Snapshotting<UIView, UIImage>.image.pullback { view in
+            let containerView = AccessibilitySnapshotView(
+                containedView: view,
+                viewRenderingMode: .renderLayerInContext,
+                markerColors: markerColors,
+                activationPointDisplayMode: activationPointDisplayMode
+            )
+
+            let window = UIWindow(frame: UIScreen.main.bounds)
+            window.makeKeyAndVisible()
+            containerView.center = window.center
+            window.addSubview(containerView)
+
+            containerView.parseAccessibility(useMonochromeSnapshot: useMonochromeSnapshot)
+            containerView.sizeToFit()
+
+            return containerView
+        }
+    }
+
+    /// Snapshots the current view using the specified content size category to test Dynamic Type.
+    ///
+    /// - parameter contentSizeCategory: The content size category to use in the snapshot
+    public static func image(
+        at contentSizeCategory: UIContentSizeCategory
+    ) -> Snapshotting {
+        return Snapshotting<UIView, UIImage>.image(
+            traits: .init(preferredContentSizeCategory: contentSizeCategory)
+        )
+    }
+
+    /// Snapshots the current view simulating the way it will appear with Smart Invert Colors enabled.
+    @available(iOS 11, *)
+    public static var imageWithSmartInvert: Snapshotting {
```

Comment:
We should add a test for this one

- [ ] Add a test for snapshotting with inverted colors

Reply:
I need to actually double check this one actually even works - I vaguely remember when I transitioned over all the APIs that this one wasn't acting as expected but been a while.

Reply:
Addressed in 6f38a7e. Looks like it's working correctly to me.